tests/Tests/600__ai_apps/670__pachyderm/670__pachyderm.robot

```diff
@@ -0,0 +1,28 @@
+*** Settings ***
+Library         SeleniumLibrary
+Resource        ../../../Resources/Page/OCPLogin/OCPLogin.resource
+Resource        ../../../Resources/OCP.resource
+Test Setup      Dashboard Test Setup
+Test Teardown   Dashboard Test Teardown
+
+*** Variables ***
+${pachyderm_container_name}     Pachyderm
+${pachyderm_appname}            pachyderm
+
+*** Test Cases ***
+Verify Inetl AIKIT Operator Can Be Installed Using OpenShift Console
```

Comment:
Test case name should be changed to Pychyderm. Here it is mentioned intel aikit

Reply:
sorry actually I was copying it and it happened by mistakenly. I am changing it.